libs/sdk-ui-dashboard/src/presentation/drill/utils/drillDownUtils.ts

```diff
@@ -16,3 +18,30 @@ export function getDrillDownAttributeTitle(drill: IDrillDownDefinition, drillEve
         )
         .map((intersectionAttributeItem) => intersectionAttributeItem.attributeHeaderItem.name)[0];
 }
+
+/**
+ * Implicit drillDown has lower priority, so needs to be removed when other drill config exists for the same attribute
+ *
+ * @internal
+ */
+export function filterDrillFromAttributeByPriority(
+    drillDefinitions: DashboardDrillDefinition[],
+): DashboardDrillDefinition[] {
+    const nonImplicitDrillDown = drillDefinitions.filter(
```

Comment:
Nitpick: Is the naming correct? These are not only drillDowns, or am I missing something? Maybe drillDefinitionsWithoutDrillDown or possibleConflictingDrillDefinitions ? I have no idea about better naming... or maybe just "drills" + comment about the reason of filtering them ? :/

Reply:
renamed and use just one filter